ReleaseProcessAutomation.Tests/Steps/InitialBranching/BranchFromReleaseStepTests.cs

```diff
@@ -34,17 +34,17 @@ public void Setup ()
   {
     _gitClient = new Mock<IGitClient>();
     _inputReader = new Mock<IInputReader>();
-    _releaseRcStep = new Mock<IReleaseRCStep>();
-    _releaseWithRcStep = new Mock<IReleaseWithRcStep>();
+    _releaseRCStep = new Mock<IReleaseRCStep>();
+    _releaseWithRCStep = new Mock<IReleaseWithRCStep>();
```

Comment:
We should consider adding `RC` to the list of allowed uppercase words and save it in the team-shared configuration. This way, everyone working on the project won't get naming squiggles for an `RC` in the name.

This option should be available using the context menu:
![image](https://user-images.githubusercontent.com/5082673/182385410-712a03ad-df56-4070-a34a-5a6707642091.png)